### PR TITLE
Code formatting

### PR DESCRIPTION
**Pull request by kkacprzak**
- code formatting with astyle,
- adding doxy headers, similar to those we use in Framework,
- removing redundant commented code and useless comments,
- no functional changes in the code,
- some documentation polishing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,78 +1,76 @@
...
 ## Required packages:
-- geant4.10.04
-- root 6.0 
-- cadmesh (https://github.com/christopherpoole/CADMesh)
-- cmake 
+- `cmake`
+- `root 6.0`
+- [`geant4.10.04`](https://github.com/Geant4/geant4)  
+  Built with cmake flags:  
+  `-DGEANT4_USE_QT=ON -DGEANT4_INSTALL_DATA=ON -DGEANT4_USE_OPENGL_X11=ON -DGEANT4_USE_GDML=ON -DGEANT4_BUILD_MULTITHREADED=ON`
```
Comment (DariaKisielewska) [inline README.md]: Code now is not multi-thread safe. I propose to remove -DGEANT4_BUILD_MULTITHREADED=ON`

File: README.md [hunk 2]
```diff
@@ -1,78 +1,76 @@
...
-- define fraction of oPs annihilations (0-1): /jpetmc/material/xad/oPsFraction
-- define fraction of pick-off processes (0-1): /jpetmc/material/xad/pickOffFraction
-- set center of the annihilation chamber: /jpetmc/run/setChamberCenter
-- for run5: define range where we expect annihilations to occur: /jpetmc/run/setEffectivePositronRange  
-
-General parameters:
-- define time range, between hits in scintillator, which will be classified as single hit: /jpetmc/detector/hitMergingTime
+User can load one of the selected geometries:  
+ - `/jpetmc/detector/loadGeomForRun runNr`  
+   - runNr=3/5/6
```
Comment (DariaKisielewska) [inline README.md]: 3/5/6/7

File: README.md [hunk 3]
```diff
@@ -1,78 +1,76 @@
...
+    - loaded `JPetBasicGeom`  
+ - `/jpetmc/detector/loadJPetBasicGeom`  
+   - 3 layers of scintillators (48,48,96)  
+   - each scintillator: 1.9x0.7x50 cm^3 wrapped in kapton foil  
+   - detector frame: loaded from CAD file  
+   - standard setup for runs 1-6  
+ - `/jpetmc/detector/loadIdealGeom`  
+   - not implemented yet  
+   - should load n-layer fully packed with scintillators barrel  
+ - `/jpetmc/detector/loadJPetExtendedGeom`
```
Comment (DariaKisielewska) [inline README.md]: ready - this flag is loading 4th layer

File: README.md [hunk 4]
```diff
@@ -1,78 +1,76 @@
...
+- simulate only pick-off process (2g & long lifetime): `/jpetmc/material/xad/pickOffOnly`  
+- define longest lifetime component: `/jpetmc/material/xad/oPslifetime`  
+- define fraction of oPs annihilations (0-1): `/jpetmc/material/xad/oPsFraction`  
+- define fraction of pick-off processes (0-1): `/jpetmc/material/xad/pickOffFraction`  
+- set center of the annihilation chamber: `/jpetmc/run/setChamberCenter`  
+- for run5: define range where we expect annihilations to occur: `/jpetmc/run/setEffectivePositronRange`  
 
+General parameters:  
+- define time range, between hits in scintillator, which will be classified as single hit:  
+`/jpetmc/detector/hitMergingTime`
```
Comment (DariaKisielewska) [inline README.md:39]: more paramaters:
/jpetmc/event/save2g  Events with registered 2g will be saved
/jpetmc/event/save3g Events with registered 3g will be saved
/jpetmc/event/printEvtStat Print how many events was generated
/jpetmc/event/printEvtFactor Define X in divisor (10^X) for number of printed event
/jpetmc/event/ShowProgress Print how many events was generated (in %)

/jpetmc/source/nema  Give nema point number to simulate (1-6)

/jpetmc/source/gammaBeam/setEnergy  setPosition setMomentum

/jpetmc/source/isotope/setShape cylinder
/jpetmc/source/isotope/setNGamma  Give number of gamma quanta to generate 1 / 2 / 3 
/jpetmc/source/isotope/setShape/cylinderRadius
/jpetmc/source/isotope/setShape/cylinderZ  For cylindrical shape - set z (half length)
/jpetmc/source/isotope/setPosition  Set position of the source

**Review by DariaKisielewska — CHANGES_REQUESTED** [review]
Minor changes requested

File: README.md (hunk 1 above)
Comment (kkacprzak) [reply README.md]: ok

File: README.md (hunk 2 above)
Comment (kkacprzak) [reply README.md]: ok

File: README.md (hunk 3 above)
Comment (kkacprzak) [reply README.md]: ok

File: README.md (hunk 4 above)
Comment (kkacprzak) [reply README.md:39]: I created new file with parameters and moved all of them there

**Comment by kkacprzak** [comment]
Changes done, ready for next review

**Review by DariaKisielewska — APPROVED** [review]

**Pull request merged**